core/domain/platform_parameter_registry.py

```diff
@@ -275,9 +276,9 @@ def load_platform_parameter_from_memcache(cls, name):
             PlatformParameter|None. The loaded instance, None if it's not found
             in cache.
         """
-        memcache_key = param_domain.PlatformParameter.get_memcache_key(name)
```

Comment:
Please remove the old get_memcache_key method if it isn’t needed anymore.

Reply:
Already removed I believe.